### DISCUSSION TITLE
Square Root?

### DISCUSSION THREAD

**Discussion by rkmfl** [antonmedv/numbr#40] [General]
First, Numbr is one of my favorite extensions, thanks so much for developing it!

Second, I've never figured out how to do Square Root operations with Numbr and I don't see it in the documentation.  Is this possible?

If it's not possible, is this something that could be added?

Thank you in advance.

**Comment by antonmedv** [comment]
Yes unfortunately the square root is not a possible right now. 

I’m working on a new version with square root and other math functions will be available

**Comment by rkmfl** [comment]
@antonmedv - Thank you, that's excellent news.

Would it be easier to rely upon a prebuilt math engine that is available open source, for example AsciiMath?  https://asciimath.org/
